src/hotspot/share/jeandle/jeandleCallVM.cpp

```diff
@@ -37,6 +37,7 @@ void JeandleCallVM::generate_call_VM(const char* name, address c_func, llvm::Fun
                                                      target_module);
   llvm_func->setCallingConv(llvm::CallingConv::Hotspot_JIT);
   llvm_func->setGC(llvm::jeandle::JeandleGC);
+  llvm_func->addFnAttr(llvm::Attribute::NoCfCheck);
```

Comment:
```suggestion
  JeandleFuncSig::setup_description(llvm_func, true /*is_stub*/);
```
Maybe we can call setup_description here. But for stub functions, we don't need to add patchable-function-entry, so we can add an argument for setup_description to indicate if we are setting up a stub function.
By this way, we can make all llvm function setup process in one place.

Reply:
Fixed